src/pylexibank/dataset_templates/lexibank_simple/lexibank_+id+.py_tmpl

```diff
@@ -0,0 +1,81 @@
+from clldutils.path import Path
```

Comment:
quick q: what does clldutils.Path give us that pathlib doesn't? is it just py2/3 compat (which we don't need anymore)?